browse-building-floors/src/main/java/com/esri/arcgismaps/sample/browsebuildingfloors/MainActivity.kt

```diff
@@ -0,0 +1,182 @@
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.browsebuildingfloors
+
+import android.content.Context
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import android.widget.TextView
+import androidx.annotation.LayoutRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import androidx.lifecycle.lifecycleScope
+import com.arcgismaps.ApiKey
+import com.arcgismaps.ArcGISEnvironment
+import com.arcgismaps.mapping.ArcGISMap
+import com.arcgismaps.mapping.floor.FloorLevel
+import com.arcgismaps.mapping.floor.FloorManager
+import com.arcgismaps.portal.Portal
+import com.arcgismaps.portal.PortalItem
+import com.esri.arcgismaps.sample.browsebuildingfloors.databinding.ActivityMainBinding
+import com.google.android.material.snackbar.Snackbar
+import kotlinx.coroutines.launch
+
+class MainActivity : AppCompatActivity() {
+
+    private val TAG = MainActivity::class.java.simpleName
+
+    // set up data binding for the activity
+    private val activityMainBinding: ActivityMainBinding by lazy {
+        DataBindingUtil.setContentView(this, R.layout.activity_main)
+    }
+
+    private val mapView by lazy {
+        activityMainBinding.mapView
+    }
+
+    private val currentFloorTV by lazy {
+        activityMainBinding.selectedFloorTV
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        // authentication with an API key or named user is
+        // required to access basemaps and other location services
+        ArcGISEnvironment.apiKey = ApiKey.create(BuildConfig.API_KEY)
+        lifecycle.addObserver(mapView)
+
+        // load the portal and create a map from the portal item
+        val portalItem = PortalItem(
+            Portal("https://www.arcgis.com/"),
+            "f133a698536f44c8884ad81f80b6cfc7"
+        )
+
+        // set the map to be displayed in the layout's MapView
+        val map = ArcGISMap(portalItem)
+        mapView.map = map
+
+        lifecycleScope.launch {
+            //load the portal item on the map
+            map.load().getOrElse {
+                showError("Error loading portal item" + it.message.toString())
+                return@launch
+            }
+
+            // load the web map's floor manager
+            val floorManager =
+                map.floorManager ?: return@launch showError("Map is not a floor aware web-map")
```

Comment:
The error message sounds a bit odd. And some terms are not commonly used such as `web-map`. 
```suggestion
                map.floorManager ?: return@launch showError("Map is not floor-aware")
```